backend/molgenis-emx2-io/src/main/java/org/molgenis/emx2/io/ImportTableTask.java

```diff
@@ -104,7 +104,7 @@ public void process(Iterator<Row> iterator, TableStore source) {
                   .map(Field::getName)
                   .collect(Collectors.joining(","));
           task.addSubTask("Found duplicate Key (" + keyFields + ")=(" + keyValue + ")").setError();
-        } else {
+        } else if (!keyValue.equals("null")) {
```

Comment:
Is it a good idea to instead check for if(keyValue.equals("null") and throw an error in that case? (just an idea, not sure if that's what you'd want here)